ceph.spec.in

```diff
@@ -48,8 +48,10 @@
 %endif
 %if 0%{?is_opensuse}
 %bcond_without libradosstriper
+%bcond_without ocf
```

Comment:
just want to check with you, so openSUSE does support the OCF resource agents?

Reply:
I don't think "support" is the right word. Instead, I would say: "there is no reason to deprive openSUSE users of this package."